sway-fmt-v2/src/items/item_abi.rs

```diff
@@ -74,12 +74,12 @@ impl CurlyBrace for ItemAbi {
             ItemBraceStyle::AlwaysNextLine => {
                 // Add openning brace to the next line.
                 line.push_str(&format!("\n{}\n", open_brace));
-                shape = shape.block_indent(1);
+                shape = shape.block_indent(4);
```

Comment:
All of these `4` literals should be behind a const or come from config

Reply:
Agreed, I was thinking that as I placed it there haha.

As for the `fn` handling, I also agree. It's sort of a big task since the rest of the formatter will rely on it. For the time being I think this could still go through, and I can work on `fn` next then implement it here. Either way is fine by me since they're both necessary (: